test/loadTest/v1_vs_v2.sh

```diff
@@ -0,0 +1,93 @@
+﻿#!/bin/bash
```

Comment:
Rather than v1_vs_v2.sh, I'd suggest: update_time_measure.sh or something like that.